app/models/concerns/solidus_stripe/money_to_stripe_amount_converter.rb

```diff
@@ -72,6 +74,24 @@ def to_solidus_amount(fractional, currency)
     end
   end
 
+  # Transforms a decimal amount into a subunit amount
+  #
+  # @param amount [BigDecimal]
+  # @param currency [String]
+  # @return [Integer]
+  def solidus_decimal_to_subunit(amount, currency)
+    Money.from_amount(amount, currency).cents
```

Comment:
nit: Given the multiplier can vary and they're not always cents we could use `fractional` instead of `cents`, the two methods are aliased but `fractional` seems less misleading.

Reply:
Makes a lot of sense. Thanks for the suggestion. Changed!